DayrSuryan/DSEthiop7.xml

```diff
@@ -255,88 +280,126 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                           </q>
+                        </item>
 
                         <item xml:id="e1">
-                           <locus target="#ivr"/>
-                           <desc>Two Arabic stamps of the library</desc>
+                           <locus target="#ir #iiv"/>
+                           <desc>Stamps of <placeName ref="INS0478DS"/>.</desc>
                            <q xml:lang="ar"/>
                         </item>
 
                         <item xml:id="e2">
-                           <locus target="#ivr"/>
-                           <desc>Arabic shelfmarks</desc>
-                           <q xml:lang="ar"/>
+                           <locus target="#ir"/>
+                           <desc>Arabic shelfmark.</desc>
+                           <q xml:lang="ar">مسلسل ٧١٢</q>
                         </item>
-
+                        
                         <item xml:id="e3">
-                           <locus target="#ivr"/>
-                           <desc>Arabic title</desc>
-                           <q xml:lang="ar">صلوات</q>
+                           <locus target="#ir"/>
+                           <desc>Arabic shelfmark.</desc>
+                           <q xml:lang="ar">٦١٨ ب طقوس</q>
                         </item>
 
                         <item xml:id="e4">
-                           <locus target="#ivr"/>
-                           <desc>Arabic scribbles (year dates).</desc>
-                           <q xml:lang="ar"/>
+                           <locus target="#ir"/>
+                           <desc>Arabic title.</desc>
+                           <q xml:lang="ar">صلوات</q>
                         </item>
 
                         <item xml:id="e5">
-                           <locus target="#ivr"/>
+                           <locus target="#ir"/>
+                           <desc>Arabic scribbles (year dates) in pencil.</desc>
+                           <q xml:lang="ar"/>
+                        </item>
+                        
+                        <item xml:id="e6">
+                           <locus target="#ir"/>
+                              <desc>Arabic note in pencil.</desc>
+                           <q xml:lang="ar">الراهب <persName ref="PRS11959Pawlos">بولس</persName></q>
+                        </item>
+
+                        <item xml:id="e7">
+                           <locus target="#ir"/>
                            <desc>Ethiopic scribbles.</desc>
                            <q xml:lang="gez"/>
                         </item>
 
-                        <item xml:id="e6">
+                        <item xml:id="e8">
                            <locus target="#89v"/>
                            <desc>Arabic scribbles of the scribe's name and year dates.</desc>
-                           <q xml:lang="ar"> الراهب <persName ref="PRS11959Pawlos">بولس
```

Comment:
of the name ...., two (four?) dates (could they be listed?). Stricktly speaking, the note does not say that he is the scribe.